lib/libunw/include/unw/unwind.h

```diff
@@ -118,4 +118,6 @@ static inline void print_stack_arm64(struct unwind_state_arm64 *state __unused,
  */
 void ftrace_map_lr(uint64_t *lr);
 
+void pauth_strip_pac(uint64_t *lr);
```

Comment:
a word of description would be nice.
/* Strip out PAuth tags from LR content if applicable */